cosmos/lib/cosmos/api/limits_api.rb

```diff
@@ -295,8 +306,10 @@ def get_limits_sets(scope: $cosmos_scope, token: $cosmos_token)
     # @param limits_set [String] The name of the limits set
     def set_limits_set(limits_set, scope: $cosmos_scope, token: $cosmos_token)
       authorize(permission: 'tlm_set', scope: scope, token: token)
-      Logger.info("Setting Limits Set: #{limits_set}")
-      LimitsEventTopic.write({ type: :LIMITS_SET, set: limits_set.to_s }, scope: scope)
+      message = "Setting Limits Set: #{limits_set}"
+      Logger.info(message)
+      LimitsEventTopic.write({ type: :LIMITS_SET, set: limits_set.to_s,
+        time_nsec: Time.now.to_nsec_from_epoch, message: message }, scope: scope)
```

Comment:
Adding the time_nsec and message to this and set_limits made it compatible with the LimitsEvent component in Vue